app/presenters/profile.rb

```diff
@@ -77,6 +77,18 @@ def mentored_tracks
         user.track_mentor
       end
 
+      def can_contribute?
```

Comment:
Everyone can contribute, it's not clear from the name what this actually represents.